app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -208,7 +209,9 @@ class DescriptionEditFragment : Fragment() {
         binding.fragmentDescriptionEditView.setEditAllowed(editingAllowed)
         binding.fragmentDescriptionEditView.updateInfoText()
 
-        if (ReleaseUtil.isPreBetaRelease && pageTitle.description.isNullOrEmpty()) {
+        if (ReleaseUtil.isPreBetaRelease && SuggestedArticleDescriptionsDialog.availableLanguages().contains(pageTitle.wikiSite.languageCode) &&
+            action == DescriptionEditActivity.Action.ADD_DESCRIPTION && pageTitle.description.isNullOrEmpty()) {
+            binding.fragmentDescriptionEditView.showSuggestedDescriptionsLoadingProgress()
```

Comment:
Would it be possible to take this logic out as a value:
```
val isSuggestionButtonEnabled= ReleaseUtil.isPreBetaRelease && SuggestedArticleDescriptionsDialog.availableLanguages().contains(pageTitle.wikiSite.languageCode) &&
            action == DescriptionEditActivity.Action.ADD_DESCRIPTION && pageTitle.description.isNullOrEmpty()
```

and use it in `updateSuggestedDescriptionsButtonVisibility()` in `DescriptionEditView,kt`? For now, the suggestion button will always be visible even if we access the article description editing from an article page.

Reply:
Done.. and we need to show suggestions even when user is coming from article .. however showing them for languages that do not have it enabled is taken care of